src/components/switchButtons/SwitchButtons.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import cx from 'classnames';
+
+import { Container } from '../Container';
+import { createBemBlockBuilder } from '../../utils';
+
+import './SwitchButtons.scss';
+
+const BUTTON_WIDTH = 239;
+
+export const SwitchButtons = ({ buttons, onSwitch }) => {
+  const [switchState, setSwitchState] = useState(buttons);
+
+  const onSwitchButton = ({ btnName, active }) => {
+    if (active) {
+      return false
+    }
+
+    setSwitchState((previousSwitchState) => 
+      previousSwitchState.map((btn) => btnName === btn.text ? ({...btn, active: true}) : ({...btn, active: false}))
+    );
+
+    onSwitch(btnName);
+  };
+
+  const getWidth = () =>  BUTTON_WIDTH * switchState.length
+
+  return (
+    <div className="switch">
+      <div className="switch__inner" style={{maxWidth: getWidth()}}>
```

Comment:
```suggestion
      <div className="switch__inner" style={{ maxWidth: getWidth() }}>
```

Is prettier not handle it?

But the issue actually again managing it in styles. You can put the width on each button and it will expand the inner width

Reply:
Currently, I haven't managed to implement it in another way. Could we temporarily merge this variant?

Reply:
@WebDevWorker @ViktorSoroka07 
I would have preferred Viktor's suggestion, but it looks like this component will be changed anyway as part of the `Pricing` page development as it will support URL integration. So let's postpone the refactoring until this improvement.